### PR TITLE
{bio}[foss/2019a] Biopython v1.73 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)
Minor refresh of Biopython-1.73

### CODE REVIEW

File: easybuild/easyconfigs/b/Biopython/Biopython-1.73-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+dependencies = [
+    ('SciPy-bundle', '2019.03'),  # for numpy
+]
+
+download_dep_fail = True
+use_pip = True
+
+# required because we're building Python packages using Intel compilers on top of Python built with GCC
+check_ldshared = True
```
Comment (Micket) [inline easybuild/easyconfigs/b/Biopython/Biopython-1.73-foss-2019a.eb]: You shouldn't need this for foss

Comment (lexming) [reply easybuild/easyconfigs/b/Biopython/Biopython-1.73-foss-2019a.eb]: A total oversight, thanks for pointing it out. Fixed.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-s1.int.private - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.14
See https://gist.github.com/cedd0fad7e9bb003173290b36bbe12f4 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4558e79d983bb67406ed587e0ac07ca0 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @lexming!

**Pull request merged**